ui/app/components/inference/InputSnippet.stories.tsx

```diff
@@ -20,333 +20,313 @@ type Story = StoryObj<typeof meta>;
 
 export const Empty: Story = {
   args: {
-    input: {
-      messages: [],
-    },
+    messages: [],
```

Comment:
This diff looks large, but the only thing changed is that props are flattened so `args` receives `messages` and `system` props directly.